operators/dds/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+	"operator": {
+		"name": "DDS Operators",
```

Comment:
Seems like we have 1:1 metadata.json files for DDS operators already, is this redundant?

If this needs further discussion can we move it to a standalone PR?

Reply:
Thanks, Tom! It’s essential to display the group of operators’ names correctly and render the README file. If you have any concerns, let’s discuss them during today’s meeting.

Reply:
Understood. Given that `metadata.json` is used to inform HoloHub CLI functionality, I'd suggest we review other mechanisms to generate the landing page that won't impact functionality. Can discuss today

Reply:
@tbirdso thanks for the review. Any remaining concern here?